pkg/drivers/kic/oci/network_create.go

```diff
@@ -156,6 +156,9 @@ func tryCreateDockerNetwork(ociBin string, subnet *network.Parameters, mtu int,
 		if strings.Contains(rr.Output(), "is being used by a network interface") {
 			return nil, ErrNetworkGatewayTaken
 		}
+		if strings.Contains(rr.Output(), "is already used on the host or by another config") {
```

Comment:
WYT of changing from `is already used on the host` to `is already in use on the host`?

Reply:
Huh? This needs to match the output of Podman, it's not up to us to change.